.tool-versions

```diff
@@ -0,0 +1 @@
+pnpm 8.15.5
```

Comment:
Hmm why do you need this? The lockfile specification in `pnpm-lock` should be enough right?

Reply:
Yeah, it's a bit tricky. 

pnpm is at 9.x right now (and actually 10.x coming soon), so we're behind on pnpm upgrades. my global pnpm is 9.x and when I opened this up in a separate cursor instance when I did pnpm install/add it updated the lock file to 9.x version. 

This file will both give folks a notice that we're still on 8.x and two that if they're using [asdf](https://asdf-vm.com/) that will be smart to ensure that we run pnpm commands on 8.x.

Likely not needed if we were on the 9.x across our repos. It's on my TODO :D